modules/charon-core/src/main/java/org/wso2/charon3/core/encoder/JSONDecoder.java

```diff
@@ -51,10 +51,7 @@
 import org.wso2.charon3.core.utils.codeutils.SearchRequest;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Avoid * imports.
Import just only the required classes

Reply:
Thanks. I changed that: https://github.com/wso2/charon/pull/379/commits/f276f8939dd7f4b86a9fd0054fc09866cd753124

Reply:
@AnuradhaSK am I ready to merge?

Reply:
@AnuradhaSK am I still missing something?